src/pages/journal/journal.ts

```diff
@@ -113,6 +161,7 @@ export class JournalPage extends BasePageComponent implements OnInit, OnDestroy
   };
 
   showError = (error: MesError): void => {
+    if (this.pageState.isPolling$) return;
```

Comment:
What would the reason for skipping the error message if it's polling?

On this note I would question that the `isPolling$` is part of the `JournalPageState`.

Reply:
Logic for whether to show the error not is now in the Effect, not the journal page.